src/utils/getPercentageOfDaysLeft.ts

```diff
@@ -1,19 +1,20 @@
 import moment from 'moment';
 
-function inputParser(input: string) {
-    const parsedDate = moment(new Date(parseInt(input, 10) * 1000));
-    return parsedDate;
-}
-
 const getPercentageOfDaysLeft = (startedOn: string, endsOn: string): number => {
-    const startDate = inputParser(startedOn);
-    const endDate = inputParser(endsOn);
+    const startDate = moment(new Date(parseInt(startedOn, 10) * 1000));
```

Comment:
can we take this piece out as a function

Reply:
I didn't understand, why this needs to be taken out  as a separate function.

Reply:
Before your changes, it was a separate function. Do you were facing any issues?